### ISSUE TITLE
Bugs found so far

### ISSUE THREAD

**Issue by Quetzals** [SEServerExtender/SEServerExtender#90]
Any new interior lights I build dont work. All interior lights built before this update work just fine.
small ship spot lights dont work right after they get a certain range from blocks in front of them.
When you spawn or respawn at medbay you instantly loss all power and have to recharge before you can do anything.

This is all I've run across so far.

**Comment by ghost** [comment]
The energy/health bug is a game bug, because i have disabled SESE at the moment, and i had the same problem

**Comment by SilentSymphony** [comment]
https://github.com/SEServerExtender/SEServerExtender/issues/89 most of the bugs are posted here

SESE is running fine no reason to disable it, However i would not leave it unattended.

**Comment by markam67** [comment]
Actually I had to abandon SESE for the last few days as every time multiple people would attempt to connect one person (usually me as my ping to the machine is like <10) and everyone else would get host has left the game when trying to connect. Once I started using the DS from keen it seemed more stable, however I would say that even though we all got connected (4 people) then there were lag related issues during game-play. I am much more likely to blame the issues on the very rough update from Keen.
